scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,531 @@
+  # BYTES_DB_URI=postgresql://bytes:<password>@localhost/bytes_db
+  - name: Store password in /etc/kat/bytes.conf
+    ansible.builtin.lineinfile:
+      path: /etc/kat/bytes.conf
+      regexp: "BYTES_DB_URI=.*"
+      line:  "BYTES_DB_URI=postgresql://bytes:{{ bytes_password }}@localhost/bytes_db"
+
+  # Create a new database and user for Bytes:
+  #
+  # sudo -u postgres createuser bytes -P
+  - name: Create Postgresql user bytes and grant privs
+    become: true
+    become_user: postgres
+    community.postgresql.postgresql_user:
+      name: bytes
+      password: "{{ bytes_password }}"
+      role_attr_flags: NOSUPERUSER
+
+
+  # sudo -u postgres createdb bytes_db
+  # sudo -u postgres psql -c 'GRANT ALL ON SCHEMA public TO bytes;' bytes_db
+  - name: Create Postgresql database katalogus_db
+    become: true
+    become_user: postgres
+    community.postgresql.postgresql_db:
+      name: bytes_db
+      owner: bytes
+      encoding: UTF-8
+      template: template0
+
+  # Insert this password into the connection string for the Mula DB in /etc/kat/mula.conf. For example:
+  # SCHEDULER_DB_DSN=postgresql://mula:<password>@localhost/mula_db
+  # Somebody renamed this to SCHEDULER_DB_URI... Le sigh...
+  - name: Store password in /etc/kat/mula.conf
+    ansible.builtin.lineinfile:
+      path: /etc/kat/mula.conf
+      regexp: "SCHEDULER_DB_URI=.*"
+      line:  "SCHEDULER_DB_URI=postgresql://mula:{{ mula_password }}@localhost/mula_db"
+
+  # Create a new database and user for Mula:
+  # sudo -u postgres createuser mula -P
+  - name: Create Postgresql user bytes and grant privs
+    become: true
+    become_user: postgres
+    community.postgresql.postgresql_user:
+      name: mula
+      password: "{{ mula_password }}"
+      role_attr_flags: NOSUPERUSER
+
+  # sudo -u postgres createdb mula_db
+  # sudo -u postgres psql -c 'GRANT ALL ON SCHEMA public TO mula;' mula_db
+  - name: Create Postgresql database mula_db
+    become: true
+    become_user: postgres
+    community.postgresql.postgresql_db:
+      name: mula_db
+      owner: mula
+      encoding: UTF-8
+      template: template0
+
+  # Create Rocky superuser and set up default groups and permissions
+  # Create an admin user for OpenKAT
+  # sudo -u kat rocky-cli createsuperuser
+  # System check identified some issues:
+  # WARNINGS:
+  # ?: (staticfiles.W004) The directory '/opt/venvs/kat-rocky/lib/python3.11/site-packages/assets' in the STATICFILES_DIRS setting does not exist.
+  # Email: root@localhost
+  # Full name: Superadmin account
+  # Password:
+  # Password (again):
+  # This password is too short. It must contain at least 12 characters.
+  # This password must contain at least 2 digits.
+  # This password must contain at least 2 upper case letters.
+  # This password must contain at least 2 special characters.
+  # In case the rules change we accept a bad password.
+  - name: Create Rocky superuser
```

Comment:
Deze taak faalde in mijn test-run

raise source.error(\"missing ), unterminated subpattern\",\r\nre.error: missing ), unterminated subpattern at position 9\r\n", "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", "rc": 1}


Maar niet helemaal duidelijk waarom